src/main/java/de/uhd/ifi/se/decision/management/jira/rest/ConsistencyRest.java

```diff
@@ -56,7 +75,71 @@ public Response discardLinkSuggestion(@Context HttpServletRequest request, @Quer
 		jsonMap.put("key", linkSuggestion.getTargetIssue().getKey());
 		jsonMap.put("summary", linkSuggestion.getTargetIssue().getSummary());
 		jsonMap.put("id", linkSuggestion.getTargetIssue().getId());
-		jsonMap.put("score", linkSuggestion.getScore());
+		jsonMap.put("score", linkSuggestion.getTotalScore());
+		jsonMap.put("results", linkSuggestion.getScore());
+
+		return jsonMap;
+	}
+
+
+	//--------------------
+	// Duplicate issue detection
+	//--------------------
+
+	@Path("/getDuplicatesForIssue")
+	@GET
+	@Produces({MediaType.APPLICATION_JSON})
+	public Response getDuplicatesForIssue(@Context HttpServletRequest request, @QueryParam("issueKey") String issueKey) {
+		Issue baseIssue = ComponentAccessor.getIssueManager().getIssueByCurrentKey(issueKey);
+		HashMap<String, Object> result = new HashMap<>();
+		DuplicateDetectionManager manager = new DuplicateDetectionManager(baseIssue, new BasicDuplicateTextDetector( ConfigPersistenceManager.getMinDuplicateLength(baseIssue.getProjectObject().getKey())));
+		try {
+			// get Issues of project
+			Collection<Long> issueKeysToCheck = ComponentAccessor.getIssueManager().getIssueIdsForProject(baseIssue.getProjectId());
+			Collection<Issue> issuesToCheck = new ArrayList<>();
+			for (Long issueId : issueKeysToCheck) {
+				issuesToCheck.add(ComponentAccessor.getIssueManager().getIssueObject(issueId));
+			}
+			// detect duplicates
+			List<DuplicateFragment> foundDuplicateFragments = manager.findAllDuplicates(issuesToCheck);
+
+			// convert to Json
+			List<Map<String, Object>> jsonifiedIssues = new ArrayList<>();
+			for (DuplicateFragment duplicateFragment : foundDuplicateFragments) {
+				jsonifiedIssues.add(this.duplicateToJsonMap(duplicateFragment));
+			}
+			result.put("duplicates", jsonifiedIssues);
+			return Response.ok(result).build();
+		} catch (GenericEntityException e) {
+			e.printStackTrace();
+			return Response.status(500).entity(e).build();
+		}
+	}
+
+
+	@Path("/discardDuplicate")
+	@POST
+	@Produces({MediaType.APPLICATION_JSON})
+	public Response DiscardDetectedDuplicate(@Context HttpServletRequest request, @QueryParam("projectKey") String projectKey, @QueryParam("originIssueKey") String originIssueKey, @QueryParam("targetIssueKey") String targetIssueKey) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'DiscardDetectedDuplicate' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)